### PR TITLE
Fixes #RHCLOUD-23858 - Switch to UBI8-minimal base image

### PR DESCRIPTION
**Pull request by dkuc**

### CODE REVIEW

**Review by kgaikwad — CHANGES_REQUESTED** [review]
Thank you @dkuc! 
Added a small query otherwise LGTM :+1:

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,15 @@
-FROM registry.redhat.io/rhel8/python-39
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
```
[the remark this answers is not in the document]
Comment (dkuc) [reply Dockerfile:1]: Not mandatory.  Feel free to change. This whole PR should just be taken as a suggestion.

**Review by patilsuraj767 — APPROVED** [review]
LGTM :+1:  I did a quick test in EE with the PR image. Everything looks good.

**Review by kgaikwad — APPROVED** [review]

**Comment by kgaikwad** [comment]
Thank you @dkuc!

**Pull request merged**